File: A2/HobbyAnimals/Animals/Tarantula.cs
namespace HobbyAnimals;

//Primary Constructor inherited from Animal class
public class Tarantula(string name, int exhilaration, char type) : Animal(name, exhilaration, type)
{
    public override void changeEx(Mood mood)
    {
        switch (mood)
        {
            case Mood.joyful:
                increaseEx(1);
                break;
            case Mood.usual:
                decreaseEx(2);
                break;
            case Mood.blue:
                decreaseEx(3);
                break;
        }
    }
}

File: A2_py/Animals/Tarantula.py
class Tarantula(Animal):
    def __init__(self, name: str, ex: int, type:str):
        self.name = name
        self.ex = ex
        self.type = type
    def changeEx(self, mood : Mood):
        match maood:
            case Mood.JOYFUL:
                self._increaseEx(1)
            case Mood.USUAL:
                self._decreaseEx(2)
            case Mood.BLUE:
                self._decreaseEX(3)
